custom_components/tuya_local/devices/gosund_wp9rtl_3_outlet_usb_child_lock_and_state.yaml

```
@@ -0,0 +1,166 @@
+name: Gosund 3-Outlet + USB Powerstrip with Child Lock and State Memory
+products:
+  - id: 000002uovo
```

And so it is -- still figuring out the different stuff I can query. Does this look more correct?

```suggestion
  - id: rccfczzprrjvkpiz
```